src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_PROPERTY_SIZE = 5;
+    private static final Color DEFAULT_COLOR = Color.RED;
+    private static final int NUMBER_OF_FIGURES = 5;
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        Color thisFiguresColor = colorSupplier.getRandomColor();
+        switch (new Random().nextInt(NUMBER_OF_FIGURES)) {
```

Comment:
in this method you write new Random 10 times. it is better to create the object as a class field